### DISCUSSION TITLE
Using nuxt v3 instead of gridsome might be a better choice

### DISCUSSION THREAD

**Discussion by sigridchen** [awdr74100/awdr74100.github.io#98] [Q&A] [answered]
[Gridsome](https://github.com/gridsome/gridsome) has been deprecated, using nuxt.js may be a better choice, or next.js of the react ecosystem.

**Comment by awdr74100** [comment, marked as answer]
I am indeed refactoring my blog using nuxt + nuxt-content and intend to deploy it on cloudflare worker, but [#2422](https://github.com/nuxt/content/issues/2422) hindered my plan.

**Reply by sigridchen** [reply]
It seems to have a lot to do with [shikiji](https://github.com/antfu/shikiji), now we can only wait.
